doc/lua_api.txt

```diff
@@ -5832,15 +5832,88 @@ This is basically a reference to a C++ `ServerActiveObject`
 * `hud_set_hotbar_selected_image(texturename)`
     * sets image for selected item of hotbar
 * `hud_get_hotbar_selected_image`: returns texturename
-* `set_sky(bgcolor, type, {texture names}, clouds)`
-    * `bgcolor`: ColorSpec, defaults to white
-    * `type`: Available types:
-        * `"regular"`: Uses 0 textures, `bgcolor` ignored
-        * `"skybox"`: Uses 6 textures, `bgcolor` used
-        * `"plain"`: Uses 0 textures, `bgcolor` used
-    * `clouds`: Boolean for whether clouds appear in front of `"skybox"` or
-      `"plain"` custom skyboxes (default: `true`)
-* `get_sky()`: returns bgcolor, type, table of textures, clouds
+* `set_sky(parameters)`
+    * `parameters` is a table with the following optional fields:
+        * `base_color`: ColorSpec, changes fog in "skybox" and "plain".
+        * `type`: Available types:
+            * `"regular"`: Uses 0 textures, `base_color` ignored
+            * `"skybox"`: Uses 6 textures, `base_color` used as fog.
+            * `"plain"`: Uses 0 textures, `base_color` used as both fog and sky.
+        * `textures`: A table containing up to six textures in the following
+            order: top, bottom, left, right, front, back.
+        * `clouds`: Boolean for whether clouds appear in front of 
+          the skybox (default: `true`)
+        * `sky_color`: A table containing the following values, alpha is ignored:
+            * `day_sky`: ColorSpec, for the top half of the `"regular"`
+              skybox during the day. (default: `#8cbafa`)
+            * `day_horizon`: ColorSpec, for the bottom half of the 
+              `"regular"` skybox during the day. (default: `#9bc1f0`)
+            * `dawn_sky`: ColorSpec, for the top half of the `"regular"`
+              skybox during dawn/sunset. (default: `#b4bafa`)
+            * `dawn_horizon`: ColorSpec, for the bottom half of the `"regular"`
+              skybox during dawn/sunset. (default: `#bac1f0`)
+            * `night_sky`: ColorSpec, for the top half of the `"regular"`
+              skybox during the night. (default: `#006aff`)
+            * `night_horizon`: ColorSpec, for the bottom half of the `"regular"`
+              skybox during the night. (default: `#4090ff`)
+            * `indoors`: ColorSpec, for when you're either indoors or 
+              underground. Only applies to the `"regular"` skybox.
+              (default: `#646464`)
+            * `sun_tint`: ColorSpec, changes the fog tinting for the sun
+              at sunrise and sunset.
+            * `moon_tint`: ColorSpec, changes the fog tinting for the moon
+              at sunrise and sunset.
+            * `tint_type`: ColorSpec, changes which mode the directional fog
```

Comment:
To be more descriptive, i suggest these field names:
fog_tint_sun
fog_tint_moon
fog_tint_type

Reply:
New field names are fine.